datalad/plugin/dlplugin_wtf.py

```diff
@@ -0,0 +1,84 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#PLUGINSYNOPSIS: provide information about this DataLad installation
+"""
+
+"""
+
+__docformat__ = 'restructuredtext'
+
+
+# PLUGIN API
+def datalad_plugin(dataset=None):
+    """Generate a report about the DataLad installation and configuration
+
+    IMPORTANT: Sharing this report with untrusted parties (e.g. on the web)
+    should be done with care, as it may include identifying information, and/or
+    credentials or access tokens.
```

Comment:
It already filters out cfg items under 'user....' and anything that has 'token' in it. Adding more modes would be easy, but not priority in this PR.